src/process/validation/rules/validateAvailableItems.ts

```diff
@@ -249,7 +249,7 @@ export const validateAvailableItems: RuleFn = async (context: ValidationContext)
             ? null
             : error;
         }
-        return values.find((optionValue) => optionValue === value) !== undefined ? null : error;
```

Comment:
Does this cover the case of, like, custom values? Should we be doing a deep equal here instead?

Reply:
@brendanbond There is a check above this line for the object value, I'm pretty sure it should cover most of the json and custom values, so this line is only triggered for primitive ones. I changed it a bit to pass both validation in case if `dataSrc=values` and if values is an array of primitives. So all the bases seem to be covered, but we can further discuss this if you have any other adjustments in mind.